--- origsrc/malaga-7.12/Makefile.in	2008-02-28 12:38:21.000000000 -0600
+++ src/malaga-7.12/Makefile.in	2008-04-17 12:04:46.375000000 -0500
@@ -112,8 +112,7 @@
 	for f in $(MALAGA_MAN1); do \
 	  $(INSTALL_DATA) $(srcdir)/$$f $(man1dir)/$$f; done
 	$(INSTALL_DIR) $(infodir)
-	for f in malaga.info*; do $(INSTALL_DATA) $$f $(infodir)/$$f; done
-	$(INSTALL_INFO) --info-dir=$(infodir) malaga.info 
+	for f in malaga.info; do $(INSTALL_DATA) $(srcdir)/$$f $(infodir)/$$f; done
 
 uninstall:
 	for f in $(MALAGA_BIN); do rm -f $(bindir)/$$f; done
